tensor2tensor/models/transformer.py

```diff
@@ -399,7 +405,8 @@ def forced_logits():
         beam_size=beam_size,
         top_beams=top_beams,
         alpha=alpha,
-        batch_size=batch_size)
+        batch_size=batch_size,
+        decode_length_decide_end=decode_length_decide_end)
```

Comment:
Let's find a better name. How about `force_decode_length` that defaults to `False`?